androidsdk/src/test/java/com/wootric/androidsdk/WootricTest.java

```diff
@@ -115,6 +115,20 @@ public void setUp() {
         assertThat(wootric.endUser.getEmailOrUnknown()).isEqualTo("nps@example.com");
     }
 
+    @Test public void setEndUserExternalId() throws Exception {
```

Comment:
If there's no other end user with `external_id = ""` it will set that value. If there's an end user that already has that value, it will raise a UniqueViolation: ERROR: